src/common/utils.tsx

```diff
@@ -84,3 +85,8 @@ export const downloadJsonDataFile = (jsonTemplate: any): void => {
 export const isWalletOption = (option: string | WalletOptions): option is string => {
   return typeof option === "string";
 };
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const isV3Document = (document: any): document is v3.OpenAttestationDocument => {
+  return document.hasOwnProperty("@context");
+};
```

Comment:
look in oa utils

Reply:
The utils only support checking of wrapped and signed documents.

Reply:
removed, all usage of this function will change to isRawV3Document in OA utils